src/app/app.component.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { AppComponent } from "./app.component";
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { DebugElement } from "@angular/core/src/debug/debug_node";
+import { By } from "@angular/platform-browser";
+
+describe('AppComponent created correctly', () => {
```

Comment:
Jasmine se acompla muy bien con algo que se llama BDD, esta es la sintaxis basica para escribir los casos de prueba en los describe y los it's
https://martinfowler.com/bliki/GivenWhenThen.html
la estructura seria algo asi en este caso:

`describe('Given a main component', () => {
     describe('When create the component', () => {
         it('Then the TODO title should be present ', () => {
              // Hacer los assert respectivos.
         })
     })
})`